tracer/test/snapshots/native-wrapper-symbols.verified.txt

```diff
@@ -0,0 +1,14 @@
+__errno_location
+__tls_get_addr
+access
+asprintf
```

Comment:
Note: no `dlysm` in here. Prior to the fix in this PR, `dlsym` was here which caused the issues